src/basic/socket-util.c

```diff
@@ -1472,3 +1472,71 @@ int connect_unix_path(int fd, int dir_fd, const char *path) {
 
         return RET_NERRNO(connect(fd, &sa.sa, salen));
 }
+
+int socket_address_parse_unix(SocketAddress *a, const char *s) {
```

Comment:
rename `a` to `ret`?

Reply:
pre-existing condition in socket_address_parse, but sure